README.md

```diff
@@ -1 +1,196 @@
+
+```Groovy
+maven {
+    url "https://maven.mozilla.org/maven2"
+}
+```
+
+That should be it for additions to the build configuration files.  You should try to do a Gradle sync now to make sure everything works.  If you have any problems, try going back through the configuration steps to make sure you got everything.
+
+## Instrument The Project With Glean
+
+### Initial Integration
+
+Now that we have added Glean to our app, we need to do a little initialization in order to get it ready to record and send data. At the top of your `MainActivity.kt` file, add the following import statement:
+
+```Kotlin
+import mozilla.components.service.glean.Glean
+```
+
+Then inside of the `onCreate()` function of the `MainActivity`, add the following 2 lines to enable and initialize Glean:
+
+```Kotlin
+Glean.setUploadEnabled(true)
+Glean.initialize(this)
+```
+
+That’s it!  The app now has Glean installed and enabled and will be able to send the baseline ping without any additional work!
+
+### Add Custom Metric
+
+Since we are instrumenting crashes with some custom metrics, we still have some work to do.  We will need to add a `metrics.yaml` file to define the metrics we will use to record our crash information and a `pings.yaml` file to define a custom ping which will give us some control over the scheduling of the uploading of the crash telemetry.
+
+In order to do this we need to make a decision.  Mainly, what metric type will we use to represent our crash data.  This could probably be argued several ways, but I have chosen to instrument it as an [event](https://mozilla.github.io/glean/book/user/metrics/event.html), simply because events capture information in a nice concise way and have a built-in way of passing additional information using the `extras` field.  So if we want to pass along the cause, as well as a few lines of description or even potentially a few lines of stack trace, events let us do that easily (with [some limitations](https://mozilla.github.io/glean/book/user/metrics/event.html#limits)).
+
+Now that we have decided what metric type we will use, we can now create our `metrics.yaml`.  Inside of the `app` folder in the Android Studio project we are going to create a new file and call it `metrics.yaml`.  Then we can add the schema definition and the metric definition to the file, so that it looks like this:
+
+```YAML
+# Required to indicate this is a `metrics.yaml` file
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+crash:
+  exception:
+    type: event
+    description: |
+      Event to record crashes caused by unhandled exceptions
+    notification_emails:
+      - crashes@example.com
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1582479
+    expires:
+      2099-01-01
```

Comment:
Since this is an example that people could potentially copy from, I'd put reasonable numbers here, to discourage anti-patterns ;)

Reply:
Good idea!